.circleci/config.yml

```diff
@@ -154,25 +155,15 @@ jobs:
     <<: *java_defaults
     steps:
       - checkout
-      - run:
-          name: Fetch Submodules
-          command: git submodule update --init --recursive
-      - run:
-          #Circle is bad about setting up your remotes for you, you must use origin/
-          command: |
-            if [ ${CIRCLE_BRANCH} != "" ] && [ ${CIRCLE_BRANCH} != "master" ] && [ $(git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) | grep api/ | wc -l | xargs) == 0 ]; then
-              echo No relevant changes on non-master branch, skipping
-              circleci step halt
-            fi
+      - run: *update_git_submodules
+      - run: *ensure_branch_has_api_changes
       - restore_cache:
           keys:
           - api-integration-cache-{{ checksum "~/workbench/api/build.gradle" }}
-          - api-integration-cache-
-      - run:
-          working_directory: ~/workbench
-          # Used to call gsutil from the circle environment.
-          command: ci/activate_creds.sh api/circle-sa-key.json
+          - api-integration-cache-`
```

Comment:
was this backtick intentional?

Reply:
No, good catch. I'm apparently too hard on Mac keyboards and they've started rebelling on me.